src/app/accounts/delete-account.ts

```diff
@@ -0,0 +1,27 @@
+import { IdentityRepository } from "@services/kratos"
+import { AccountsRepository, UsersRepository } from "@services/mongoose"
+
+export const deleteAccount = async ({
```

Comment:
should we rename to `markAccountForDeletion`? Because the account won't be deleted immediately